test/e2e/page-objects/surname/enterSurname.js

```diff
@@ -2,9 +2,11 @@
 const { validateSurname } = require('app/assets/locale/en');
 const paths = require('paths');
 
+const waitTime = 10;
+
 function enterSurnameAndSubmit(surname) {
   const I = this;
-
+  I.waitForElement('#surname', waitTime);
```

Comment:
if we add this way - this function has been called on almost all the tests- it will wait for all the tests 10 sec